recipes/pysot/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "pySOT" %}
+{% set version = "0.1.36" %}
+{% set sha256 = "043655621466d89328cac80a5c87f050c4b4708a79ec29b5912b79c76393504a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
We are moving towards `script: python -m pip install --no-deps --ignore-installed .` (docs will be updated soon)

Reply:
Nice to know, makes perfect sense. 👍